### PR TITLE
WIP: More tests

### PR DESCRIPTION
**Pull request by abieler**

### CODE REVIEW

**Comment by Evizero** [comment]
~~since I made a squash merge of #2 you may need to `git rebase`. sorry about that~~

**Comment by Evizero** [comment]
or I'll just roll back master. Carry on I'll take care of this

edit: done

**Comment by abieler** [comment]
LOL just did the rebase before I saw you edited your post..

**Comment by Evizero** [comment]
that shouldn't be a problem since current master is an ancestor

**Comment by Evizero** [comment]
sorry for making this more complicated still

**Comment by abieler** [comment]
If tests pass, ready to merge from my side. The last commit was to remove left overs from the "one-hot" discussion.

File: test/tst_fixedrangescaler.jl.orig [hunk 1]
```diff
@@ -0,0 +1,302 @@
+X = collect(Float64, reshape(1:40, 10, 4))
```
Comment (Evizero) [inline test/tst_fixedrangescaler.jl.orig]: This file seems to be an artefact from the rebase and should be deleted before merging

**Comment by abieler** [comment]
There are 2 depwarns left for v0.6, but from the testing part, not the actual module.
Tried to get rid of them with compat but failed miserably...
Otherwise ready to merge I d say.

**Comment by Evizero** [comment]
I'll look into it. I reserved some time later today for github related things and this PR is on the top of my list. Thanks for putting so much effort into this. This package is turning into something really useful really quickly.

**Pull request merged**